src/spikeinterface/exporters/to_phy.py

```diff
@@ -181,9 +181,19 @@ def export_to_phy(
     # export templates/templates_ind/similar_templates
     # shape (num_units, num_samples, max_num_channels)
     templates_ext = sorting_analyzer.get_extension("templates")
-    templates_ext is not None, "export_to_phy need SortingAnalyzer with extension 'templates'"
+    if templates_ext is None:
+        templates_ext = sorting_analyzer.get_extension("fast_templates")
+        if templates_ext is not None and template_mode != "average":
+            raise ValueError(
+                "export_to_phy with SortingAnalyzer with extension 'fast_templates' can only work with template_mode='average'"
+            )
+        dense_templates = templates_ext.get_templates(unit_ids=unit_ids)
+    else:
+        dense_templates = templates_ext.get_templates(unit_ids=unit_ids, operator=template_mode)
+    assert (
+        templates_ext is not None
+    ), "export_to_phy requires SortingAnalyzer with either extension 'templates' or 'fast_templates'"
```

Comment:
I was thinking about this and templates_ext cannot be None, because if it is None above the templates_ext.get_emplates will fail? Or am I misreading this? It seems to me that it would be better to do a check for none after line 185. Something like
```python
if templates_ext is None:
    raise ValueError('Must have either calculated templates or fast_templates')
```

That way regardless of if the `|` notation is accepted you have the check in place.

Reply:
True, nice catch!

But as you said, I'm waiting for the `|` PR to be merged, to re-write this more cleanly :)